src/components/Typography/Text.tsx

```diff
@@ -33,6 +37,7 @@ export const Text = ({
   color = 'default',
   variant,
   leading,
+  useDefaultTracking = font === 'druk' || font === 'druk-wide',
```

Comment:
Adding this prop because we don't want the tighter letter spacing for headings (in our base CSS) when using the Druk fonts.